homework-04/routes/authentication.js

```diff
@@ -0,0 +1,173 @@
+import express from 'express'
+import jwt from 'jsonwebtoken'
+import passport from 'passport'
+import LocalStrategy from 'passport-local'
+import FacebookStrategy from 'passport-facebook'
+import TwitterStrategy from 'passport-twitter'
+import config from '../config/config.json'
+import User from '../models/User'
+
+const router = express.Router()
+
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy
+
+const usersDb = User.getUsersDB()
+const secret = config.tokenSecret
+
+router.post('/local', (req, res) => {
+  if (req.parsedQuery && req.parsedQuery.email) {
```

Comment:
don't use query for `post` requests, especially for credentials

Reply:
fixed